gt/train/ONB_ibn_19110701_009/ONB_ibn_19110701_009.tif_tl_3.gt.txt

```diff
@@ -1 +1 @@
-Innsbrucker Nachrichten
\ No newline at end of file
+Innſbrucker Nachrichten
```

Comment:
https://github.com/UB-Mannheim/AustrianNewspapers/tree/master/gt/train/ONB_ibn_19110701_009/ONB_ibn_19110701_009.tif_tl_3.png is a bad image, but looks more like "Innsbrucker".